app/assets/stylesheets/pageflow/editor/base.scss

```diff
@@ -15,6 +15,7 @@
 @import "./indicators";
 @import "./disabled_atmo_indicator";
 @import "./unsupported_browser_hint";
+@import "./change_theme";
```

Comment:
Should also be inside `.editor` block to ensure no styles can apply inside entry preview. `ChangeThemeDialog` aleady has the `editor` CSS class.